react/components/FilterSidebar.js

```diff
@@ -164,15 +164,7 @@ const FilterSidebar = ({
 
     const facetsToRemove = [...selectedFacets, ...selectedRest]
 
-    if (updateOnFilterSelectionOnMobile && preventRouteChange) {
-      navigateToFacet(facetsToRemove, preventRouteChange)
-      setClearPriceRange(true)
-
-      return
-    }
-
-    setClearPriceRange(true)
-    setFilterOperations(facetsToRemove)
```

Comment:
Yes 👍 
here is an workspace

https://clearall4--storecomponents.myvtex.com/apparel---accessories/

I have different props combinations on `clearall`, `clearall2`, `clearall3`, `clearall4` that you can test too.

Reply:
thanks! I guess it's not working properly when `updateOnFilterSelectionOnMobile` is `false`/`undefined`.
If I select some filters and try to clear them before applying my selection, it doesn't clear anything, even though it looks like it's clearing because the page is reloaded. 
It only clears the filters that were already applied

https://github.com/vtex-apps/search-result/assets/20840671/2ae608d1-91b8-4ee2-b147-f0397a464156

Reply:
Could you try it again? I was missing the preventRouteChange prop

Reply:
Done! 

https://clearall--storecomponents.myvtex.com/

as priceRange is not controlled via filterOperations I had to run a navigateTofacet at the end. It will behave the same as setClearPriceRange(true), but more readable

I also notice a bug when the filters are not applied yet and the user changes the priceRange. But it is happening on master too. I will create a Jira card to change the PriceRange to respect the `filterOperations` when it is on mobile